e2e/js_image/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "image",
+    "dependencies": {
+        "chalk": "^4.0.0"
```

Comment:
would be nice to have a README.md here explaining what the layers in the resulting image will look like. In particular do npm deps go into their own layer beneath the application code, to speed up rebuilds?